gerritbot/examples/gerritbot-console.rs

```diff
@@ -56,10 +141,19 @@ fn main() {
         .module("gerritbot")
         .module("gerritbot_gerrit")
         .timestamp(stderrlog::Timestamp::Second)
-        .verbosity(if args.verbose { 5 } else { 2 })
+        .verbosity(match (args.quiet, args.verbose) {
+            (true, _) => 0,      // ERROR
+            (false, false) => 2, // INFO
+            (_, true) => 4,      // TRACE
+        })
         .init()
         .unwrap();
 
+    if let Some(working_directory) = args.working_directory.as_ref() {
```

Comment:
is it really useful? I mean, sounds a bit frivolous to me.

Reply:
Yes, so you can `cargo run --example gerritbot-console -- -C <tempdir>` from the behave test suite and ensure the `state.json` isn't reused.